README.md

```diff
@@ -105,6 +105,7 @@ The table below describes all the available configuration values for Waggle Danc
 | `disconnect-connection-delay`     | No         | Idle metastore connection timeout. Default is `5` |
 | `disconnect-time-unit`            | No         | Idle metastore connection timeout units. Default is `MINUTES` |
 | `database-resolution`             | No         | Controls what type of database resolution to use. See the [Database Resolution](#database-resolution) section. Default is `MANUAL`. |
+| `status-polling-delay`            | No         | Milliseconds. Controls the delay that checks metastore status and updates long running connections of any status change. Default is `300 000` (every 5 minutes). 
```

Comment:
300 000 -> 300,000

Reply:
Done :)

Reply:
hang on, let's do `300000` please as that is a valid entry. `300,000` is not. I don't think thousand separators are supported (I could be wrong). But since different countries use different thousand separators as well maybe not having one is the best option. I know we write English just think it simpler not having one. There is the human readable `every 5 minutes` already to clarify.

Reply:
Yeah. I am fine with `300000`.

Reply:
Actually, yeah, you're right. The config wouldn't have comma.